### ISSUE TITLE
Error when resultBundle path has spaces

### ISSUE THREAD

**Issue by sneha-je** [TitouanVanBelle/XCTestHTMLReport#153]
Ex: xchtmlreport -r "/Users/Spaces in Path/test result.xcresult"
When the above command is executed, the following error is seen
```
Warning: Can't find invocation record for : /Users/Spaces in Path/test result.xcresult
Error: An error has occured while creating the report. Error: Error Domain=NSCocoaErrorDomain Code=512
```

**Comment by kwanhiuhong** [comment]
yeah i got the same issue as well.

**Comment by chrisballinger** [comment]
This should be resolved by #165
